module/Finna/src/Finna/View/Helper/Root/RecordLinker.php

```diff
@@ -147,19 +149,42 @@ public function getEmbeddedVideoUrl($url)
      */
     public function related($link, $source = DEFAULT_SEARCH_BACKEND)
     {
+        $driver = $this->getView()->plugin('record')->getDriver();
```

Comment:
Tämä on muuten vähän hurja, koska record-pluginissa driver voi olla oikein tai olla olematta ihan riippuen siitä, onko record-pluginia kutsutta tätä ennen ja millä tietueella. Mutta koska näin on ennenkin tehty, niin toivotaan, että toimii...